ee/clickhouse/models/person.py

```diff
@@ -18,12 +18,11 @@
 )
 from ee.kafka_client.client import ClickhouseProducer
 from ee.kafka_client.topics import KAFKA_PERSON, KAFKA_PERSON_UNIQUE_ID
-from posthog import settings
 from posthog.models.person import Person, PersonDistinctId
 from posthog.models.utils import UUIDT
 from posthog.utils import is_clickhouse_enabled
 
-if settings.EE_AVAILABLE and is_clickhouse_enabled():
+if is_clickhouse_enabled():
```

Comment:
This is a cleanup as `is_clickhouse_enabled` already checks if `settings.EE_AVAILABLE` is `True`